app/main.py

```diff
@@ -2,0 +1,39 @@
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable shared across all instances of `Animal`. This can lead to issues if not managed carefully, especially in a multi-threaded environment. Consider using instance variables or other mechanisms to manage the state of each animal.